src/Factory/OrderItemOptionFactory.php

```diff
@@ -63,8 +65,9 @@ public function createForOptionAndValue(
             /** @var ProductInterface $product */
             $product = $orderItem->getProduct();
 
+            $contextChannel = $this->channelContext->getChannel();
             /** @var ChannelInterface $channel */
-            $channel = $order->getChannel();
+            $channel = $order ? $order->getChannel() : $contextChannel;
```

Comment:
@t-n-y how do you get orderItems without order? I don't see a case where order wouldn't be set on an orderItem

Reply:
I don't know neither. I don't understand how it is possible.
i opened an issue with my probleme [https://github.com/Brille24/SyliusCustomOptionsPlugin/issues/146](https://github.com/Brille24/SyliusCustomOptionsPlugin/issues/146)